lib/blog/authenticable.ex

```diff
@@ -9,7 +9,7 @@ defmodule Blog.Authenticable do
     end
   end
 
-  def sign_out_user(conn, %{ id: id }) do
+  def sign_out_user(conn, %{ id: _id }) do
```

Comment:
Good catch!
I think we could get rid of the hash destructuring and just use `_` like this `def sign_out_user(conn, _) do`